ch_tools/monrun_checks/utils.py

```diff
@@ -0,0 +1,66 @@
+from ch_tools.common.clickhouse.client.clickhouse_client import (
+    ClickhousePort,
+    clickhouse_client,
+)
+
+
+def execute_query_client(
```

Comment:
How about moving this function to `ClickhouseClient` as member function `query_json_data()`
We can use it also in `replication_lag.py`